explainer.md

```diff
@@ -931,15 +931,22 @@ const inboxRegistration = await navigator.serviceWorker.register(
 
 The API for creating buckets and specifying / querying policies wouldn't change.
 
-TODO: Explain why this is worse than the main decision.
+This alternative was rejected because of worse ergonomics for web developers. 
+Web developers will have to specify bucket names in the options per storage API
+creating more opportunity for error by either forgetting to specify a name
+or typos in the bucket name.
```

Comment:
or typos -> or having typos?

(Parallel structure -- the other element in the list is "forgetting ...", which is a verb.)